pages/en/lb3/Where-filter.md

```diff
@@ -185,7 +185,7 @@ This table describes the operators available in "where" filters. See [Examples]
 
 | Operator  | Description|
 | ------------- | ------------- |
-| = | Equivalence. See [examples](#equivalence) below.|
+| eq | Equivalence. See [examples](#equivalence) below.|
```

Comment:
From my understanding, `=` maps to the `=` in `/weapons?filter[where][name]=M1911`. 

And I just tested on my end with LB4 app, I found that there are some issues with Node API and REST.

For Node, for filtering **numbers**, both 
`{where: {id: 1}}` and `{ where: {id:{'eq':1}} }` work.
But for filtering strings, none of them work:
`{where: {content: 'foooo'}}` and `{ where: {content:{'eq': "foooo"}} }`
( it returns all instances).


For REST, it has the same issue:
`http://localhost:3000/comments?filter[where][content]=foooo`returns all instances, while `http://localhost:3000/comments?filter[where][id]=1` works as expected.

Reply:
Sorry I didn't spot this before. I tested `where` with id so it seemed work to me :(

Reply:
@agnes512 Thank you for verifying it with a working app 👍 

> From my understanding, = maps to the = in /weapons?filter[where][name]=M1911.

Not really, `eq` here is one of the comparison operators among the list of [JS syntax operators](https://github.com/strongloop/loopback.io/blob/2d1a40d8b9c17a52c59120efd6dc2c0681c728a3/pages/en/lb3/Where-filter.md#operators). `=` is the [REST syntax](https://github.com/strongloop/loopback.io/blob/2d1a40d8b9c17a52c59120efd6dc2c0681c728a3/pages/en/lb3/Querying-data.md#rest-syntax) in url.

According to your observation, `eq` may have bug when the data is in type string, but based on my experience, it's the correct JS syntax, `=` is definitely not correct : p
We can create new issues to fix the problem you find. WDYT?

Reply:
Agree, besides the bug, I think we should make it clear that `=` is for REST and `eq` for Node.

Reply:
@agnes512 cool. The rest syntax is documented here https://github.com/strongloop/loopback.io/blob/2d1a40d8b9c17a52c59120efd6dc2c0681c728a3/pages/en/lb3/Querying-data.md#rest-syntax
https://github.com/strongloop/loopback.io/pull/964/commits/e0d26a5eebea08b786d89edc2ed11ff379f4bc3f